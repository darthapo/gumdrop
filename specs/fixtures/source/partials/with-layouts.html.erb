<p><%= render 'user', layout:'wrap' %></p>