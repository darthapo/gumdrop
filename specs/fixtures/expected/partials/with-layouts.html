<body class="site"><p><div>User</div></p></body>